cli/azd/pkg/project/service_target_appservice.go

```diff
@@ -52,7 +52,6 @@ func (st *appServiceTarget) Package(
 ) *async.TaskWithProgress[*ServicePackageResult, ServiceProgress] {
 	return async.RunTaskWithProgress(
 		func(task *async.TaskContextWithProgress[*ServicePackageResult, ServiceProgress]) {
-			task.SetProgress(NewServiceProgress("Compressing deployment artifacts"))
```

Comment:
Why are you removing this?

Reply:
There are race conditions when we set progress status for operations that takes less time than the refresh-rate from the spinner frequency.

While I was running tests locally, this operation (compress artifacts) was completed before the console was able to handle the progress-status change.